render.go

```diff
@@ -324,7 +331,7 @@ func (r *Render) renderDiagnostic(word string) {
 	}
 }
 
-func (r *Render) renderLine(line string, lexer *Lexer, diagnostics []lsp.Diagnostic) {
+func (r *Render) renderLine(line string, lexer *Lexer, diagnostics []lsp.Diagnostic, documentPos int) {
```

Comment:
it seems like documentPos is not used in the function?

Reply:
Good catch! Removed it